client/state/help/ticket/test/reducer.js

```diff
@@ -0,0 +1,86 @@
+/**
+ * External dependencies
+ */
+import { assert } from 'chai';
+
+/**
+ * Internal dependencies
+ */
+import reducer from '../reducer';
+import { dummyConfiguration } from './test-data';
+
+import {
+	HELP_TICKET_CONFIGURATION_REQUEST,
+	HELP_TICKET_CONFIGURATION_REQUEST_SUCCESS,
+	HELP_TICKET_CONFIGURATION_REQUEST_FAILURE,
+	HELP_TICKET_CONFIGURATION_DISMISS_ERROR,
+} from 'state/action-types';
+
+describe( 'ticket-support/configuration reducer', () => {
+	it( 'should default to the expected structure', () => {
+		const defaultState = reducer( undefined, {} );
+
+		assert.deepEqual( defaultState, {
+			isReady: false,
+			isRequesting: false,
+			isUserEligible: false,
+			requestError: false,
+		} );
+	} );
+
+	it( 'should set isRequesting to true', () => {
+		const state = reducer( undefined, {
+			type: HELP_TICKET_CONFIGURATION_REQUEST,
+		} );
+
+		assert.isTrue( state.isRequesting );
+	} );
```

Comment:
Certainly doesn't hurt to have more tests, but these simple uses of `createReducer` are pretty much covered by the `createReducer` tests.